packages/commons/README.md

```diff
@@ -21,6 +21,30 @@ This package exists primarily to help .NET tool developers integrate `axe-core`
 * [Deque.AxeCore.Playwright](../playwright/README.md) in combination with [Playwright for .NET](https://playwright.dev/dotnet/)
 * [Deque.AxeCore.Selenium](../selenium/README.md) in combination with [Selenium](https://www.selenium.dev/)'s [C# Selenium.WebDriver package](https://www.nuget.org/packages/Selenium.WebDriver)
 
+
+## `CachedContentDownloader`
+
+Allows cacheing of downloaded external resources.
+
+## `ContentDownloader`
+
+Provides access to content from specific resources using a provided URL.
+## `EmbeddedResourceAxeProvider`
```

Comment:
I don't think "an embedded axe resources for accessibility" is clear enough to be meaningful for a user. I'd suggest something like this, with an introduction to the concept and intended usage of an "Axe script provider" generally and more specifics about exactly what axe script you get if you ask for the embedded one:

```markdown
## Axe script providers

The `IAxeScriptProvider` interface is suitable for use as an option in an API for running an `axe-core` scan of a page. It specifies a single method, `GetScript()`, which returns a string containing JavaScript code suitable for injecting into a running page.

This library provides two `IAxeScriptProvider` implementations:

### `BundledAxeScriptProvider`

`BundledAxeScriptProvider` provides a bundled copy of [`axe-core`](https://github.com/dequelabs/axe-core) which is included with the library. The included version will match the major and minor version of the library, but may not match the patch version.

This script provider is suitable as a default option for users that don't wish to explicitly override the version of `axe-core` to use.

```csharp
IAxeScriptProvider axeScriptProvider = new BundledAxeScriptProvider();
axeScriptProvider.GetScript(); // a string containing the contents of the bundled copy of axe.min.js
\```

### `FileAxeScriptProvider`

`FileAxeScriptProvider` is an alternate option for users that wish to provide their own separate implementation of `axe-core`, usually for the purposes of pinning to a specific `axe-core` version independently of this library's version.

```csharp
new FileAxeScriptProvider("./path/to/axe.min.js");
axeScriptProvider.GetScript(); // synchronously reads the contents of file ./path/to/axe.min.js
\```

```

Reply:
Updated